build-parent/pom.xml

```diff
@@ -21,7 +21,7 @@
         <!-- These properties are needed in order for them to be resolvable by the generated projects -->
         <!-- Quarkus uses jboss-parent and it comes with 3.8.1-jboss-1, we don't want that in the templates -->
         <compiler-plugin.version>3.11.0</compiler-plugin.version>
-        <kotlin.version>1.8.22</kotlin.version>
```

Comment:
I feel like this change needs to happen in more places...  when I search for 1.8.22, I get 6 hits.